looking4mage/frontend
# Dockerize

What
----

We're sanitizing the webpack configuration, making the HTML being part of the build as oppose to magic file, removing eclint as it's just not useful if we all use .editorconfig and lint anyway, adding NGINX config and Dockerfile for building an artifact.

How to review
-------------

- Code review
- Run `npm run start`
- Expect dev to work
- Run `npm run build`
- Expect `dist` to be populated with compiled files
- Run `docker build -t looking4mage/frontend:latest .`
- Expect it to succeed
- Run `docker run -p 8080:80 looking4mage/frontend:latest`
- Visit http://localhost:8080/ and expect application to be there with in app routing
